recipe/patches/0006-Fix-crosscompiling-LLVM-tools.patch

```diff
@@ -1,6 +1,17 @@
---- CMakeLists.txt.old  2020-08-07 20:15:39.605174899 +0000
-+++ CMakeLists.txt      2020-08-07 20:20:01.383910098 +0000
-@@ -111,6 +111,16 @@
+From 6df7fb48f7d03ed28f3ecb64684b41cb86c0d237 Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
```

Comment:
Can you use my name and email address here?

Reply:
Oh, sorry. The reattribution was not intentional.